assets/scripts/index.js

```diff
@@ -0,0 +1,81 @@
+const score = document.querySelectorAll('.score')
+const totalScore = document.querySelectorAll('.total-score')
+const alert = document.querySelector('#alert')
+const btn = document.querySelector('.btn')
+const btn1 = document.querySelector('.btn1')
+let count = 0
+let totalScores = 0
+btn1.addEventListener('click', function () {
+  reset()
+})
+
+btn.addEventListener('click', function () {
+  btn.style = 'animation: none;'
+  rollBall()
+})
+
+function reset () {
+  window.location.reload()
+}
+
+function rollBall () {
+  for (let i = 0; i < score.length; i++) {
+    count++
+    btn.style = 'animation: rotation 0.1s;'
+  }
+
+  let standingPins = 10
+  const pinsHitFirst = getRandomPin(standingPins)
+  standingPins = standingPins - pinsHitFirst
+  const pinsHitSecond = getRandomPin(standingPins)
+  const totalHits = pinsHitFirst + pinsHitSecond
+  totalScores += totalHits
+  btn.style = 'animation: none;'
+  if (count === 10) {
```

Comment:
This just hurts my eyes, can we do switch instead. So far so good, though I don't see how you handle strikes and no hits or maybe I haven't been able to get those from the getRandomPin yet